src/lib/utilities/is-event-type.ts

```diff
@@ -116,6 +116,8 @@ export const eventTypes = [
   'UpsertWorkflowSearchAttributes',
   'WorkflowExecutionUpdateAccepted',
   'WorkflowExecutionUpdateCompleted',
+  'WorkflowExecutionUpdateRejected',
+  'WorkflowExecutionUpdateRequested',
```

Comment:
Do we want to add these to `all-event-types.json` and add tests to `get-group-id.test.ts` as well?

Reply:
Yup good call. Going to add those just as soon as I can figure out how to actually run a workflow with this new type (it's on a forked branch using a in-progress temporal server version).